src/workspaces.c

```diff
@@ -7,12 +7,12 @@
 #include <stdlib.h>
 #include <string.h>
 #include <strings.h>
+#include "buffer.h"
```

Comment:
This seems unrelated to the PR

Reply:
Previously `buffer.h` was included indirectly via `ssd.h`, now that's no longer the case so we need to include it here to avoid compiler errors.